src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -222,6 +223,18 @@ def remove_snapshot(self, snapname):
         snappath = self.snapshot_path(snapname)
         rmsnap(self.fs, snappath)
 
+    def snapshot_info(self, snapname):
```

Comment:
might just be generic enough to have this in `SubvolumeBase` class?

Reply:
SubvolumeBase doesn't have snapshot_create/snapshot_delete, so I thought it doesn't make sense to add it in SubvolumeBase. Does it?